packages/components/src/Enumeration/Items/Item/Item.component.js

```diff
@@ -21,10 +21,11 @@ function itemClasses(isSelected) {
 	});
 }
 
-function itemLabelClasses() {
+function itemLabelClasses(item) {
```

Comment:
You don't need to pass `item` to the method, you can directly pass `item.className` inline in the upper scope